### ISSUE TITLE
RT-AX58U is same as RT-AX3000 

### ISSUE THREAD

**Issue by jata1** [ttgapers/cakeqos-merlin#59]
Can you please add RT-AX3000 as a supported device to the cake-qos.sh script? The AX3000 is the same as AX58U just different model name

**Comment by ttgapers** [comment]
Hi @Adamm00 - is this request a possibility?

**Comment by Adamm00** [comment]
> Can you please add RT-AX3000 as a supported device to the cake-qos.sh script? The AX3000 is the same as AX58U just different model name

What is the output of;

```
[ -z "$(nvram get odmpid)" ] && model="$(nvram get productid)" || model="$(nvram get odmpid)"
echo "$model"
```

**Comment by jata1** [comment]
thanks for helping. Much appreciated. The output is...

RT-AX3000

**Comment by Adamm00** [comment]
Fixed in a82e9a6d1e93a2a596c9703dc14bd30528ae2cd6
